turbinia/turbiniactl.py

```diff
@@ -717,6 +732,29 @@ def main():
   elif args.command == 'listjobs':
     log.info('Available Jobs:')
     client.list_jobs()
+  elif args.command == 'DumpGCSLogs':
+    if not config.GCS_OUTPUT_PATH:
+      log.error('GCS storage must be enabled in order to use this.')
+      sys.exit(1)
+    if not os.path.isdir(args.output_dir):
+      log.error('Please provide a valid directory path.')
+      sys.exit(1)
+    gcs_file = '**{}**'.format(args.blob_name)
```

Comment:
I guess the asterisks here would be used for wildcard searches?

Also, can we update the format string specifier to match the rest of the file (e.g. `{0:s}`).